test/IntegrationTests/Helpers/EnvironmentTools.cs

```diff
@@ -92,6 +92,17 @@ public static string GetPlatform()
         return RuntimeInformation.ProcessArchitecture.ToString();
     }
 
+    public static string GetPlatformDir()
+    {
+        return RuntimeInformation.ProcessArchitecture switch
+        {
+            Architecture.X86 => "x86",
+            Architecture.X64 => "x64",
+            Architecture.Arm64 => "ARM64",
```

Comment:
why the different casing here?

Reply:
Not to cause (weird) issues with Visual Studio. I have no idea why they defined ARM64 all uppercase in $(AvailablePlatforms).